en/messages.json

```diff
@@ -328,6 +346,9 @@
     "popupPhoneMaskingPromoBody": {
         "message": "With phone number masking, you can now create a phone number mask that helps you get texts and calls without revealing your true number."
     },
```

Comment:
This version is for non-premium users.
<img width="156" alt="image" src="https://user-images.githubusercontent.com/13066134/190509616-49ad1c41-906b-4091-8a7d-52b3dbdc34e1.png">